### PR TITLE
Fix for Gaze API

### PR DESCRIPTION
**Pull request by gdamron**
Lazily asks the user for permission to track eye gaze

### CODE REVIEW

File: Runtime/Source/Player/Element/Gaze/GlanceController.cs [hunk 1]
```diff
@@ -32,7 +32,13 @@ public GlanceController()
         {
             _CoordinateSystem = Marshal.GetObjectForIUnknown(WorldManager.GetNativeISpatialCoordinateSystemPtr()) as SpatialCoordinateSystem;
         }
-        
+
+        /// <inheritdoc />
+        public void RequestAppPermission()
+        {
+            _ = EyesPose.RequestAccessAsync();
```
Comment (jsundra) [inline Runtime/Source/Player/Element/Gaze/GlanceController.cs:39]: `_` snuck in from testing? Or is this some fancy thing in the latest C# spec that wouldn't throw in a compilation error?

Comment (gdamron) [reply Runtime/Source/Player/Element/Gaze/GlanceController.cs:39]: fancy C# thing 🙂

**Comment by gdamron** [comment]
Doesn't hurt performance, but that doesn't mean we shouldn't check whether it's supported or previously authorized. At the very least, it would allow us to short circuit the processing.

**Comment by gdamron** [comment]
Double checked, and `RequestAccessAsync` is the only way to check whether permission has been granted. For now, I'll add a check for `IsSupported` as well

**Pull request merged**